isthisstockgood/DataFetcher.py

```diff
@@ -42,17 +43,17 @@ def fetchDataForTickerSymbol(ticker):
   # Make all network request asynchronously to build their portion of
   # the json results.
   data_fetcher.fetch_msn_money_data()
-  data_fetcher.fetch_yahoo_finance_analysis()
```

Comment:
What do you think of also trying to fetch the yahoo one, and preferring that if it exists.

The tradeoff is one additional network request, but then we could use Yahoo (if they hopefully ever bring it back). And in the meantime fallback to Zacks.

Reply:
Sure, updated!